### PR TITLE
#44- Key Manager unit testing and #57-Extract Utility functions from KeyManager

### PR DESCRIPTION
**Pull request by raularagonez**

### CODE REVIEW

File: test/e2e/keyfile.js [hunk 1]
```diff
@@ -1,18 +1,12 @@
...
-/**
- * TODO: RA - include all these unit tests
- * Console logs are commented to have a clean result output.
- * Need to have a logging mechanisms with a level toggle when debugging.
- */
+const Requests = require("../../src/modules/Requests");
 
 describe("Keyfile", () => {
   const password = "encryption_password";
   let publicKey = "";
```
Comment (scasplte2) [inline test/e2e/keyfile.js:6]: Is this used anywhere? Am I just missing it?

File: test/it/module-keymanager.js [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+            // Initialization vector size in bytes
+            ivBytes: 16,
+
+            // Private key size in bytes
+            keyBytes: 32,
+
+            // Key derivation function parameters
+            scrypt: {
+              dkLen: 32,
+              n: Math.pow(2, 12), // cost (as given in bifrost)
```
Comment (scasplte2) [inline test/it/module-keymanager.js:37]: might just want to change the comment here so it isn't confusing

File: test/e2e/keyfile.js (hunk 1 above)
Comment (raularagonez) [reply test/e2e/keyfile.js:6]: Yes, its used in line 9. "new Requests()"

**Review by scasplte2 — APPROVED** [review]
👍

**Pull request merged**